example/petstore.json

```diff
@@ -907,7 +907,6 @@
         "Pet": {
             "type": "object",
             "required": [
-                "name",
```

Comment:
Apparently the official petstore server returns null for names sometimes. Since swagger-to-graphql will do strict null checking this breaks the example. Making the field optional fixes the example.